doc/deployment-on-arbitrary-cluster.md

```diff
@@ -0,0 +1,72 @@
+# Deployment on Arbitrary Cluster
+
+In this guide we will cover installation of Multus and
+ovs-cni on your arbitrary cluster.
+
+# Requirements
+
+## A Cluster
+
+This guide requires you to have your own Kubernetes cluster. If you
+don't have one and just want to try Kubemacpool out, please refer to the [deployment
+on local cluster](deployment-on-local-cluster.md) guide.
+
+# External cluster
+
+To deploy Kubemacpool on an external cluster not provided by kubevirtci we can do
+the following steps:
+- [Deploy kubevirt and Multus](#Deploy kubevirt and Multus)
```

Comment:
Can we leave this step out for the user to figure out? Providing a specific guide for arbitrary does not feel maintainable. Maybe we could just list it as dependencies and then show how to deploy released KMP and one built locally

Reply:
I agree. will leave links for kubevirt & multus quickstart guides.